packages/gatsby-remark-prismjs/src/highlight-code.js

```diff
@@ -14,6 +15,13 @@ module.exports = (language, code, lineNumbersHighlight = []) => {
       if (language === `none`) {
         return code // Don't escape if set to none.
       } else {
+        if (!notSupportedLanguages.includes(language)) {
+          console.warn(
+            `unable to find prism language '${language}' for highlighting.`,
+            `applying generic code block`
+          )
+          notSupportedLanguages.push(language)
```

Comment:
```suggestion
          unsupportedLanguages.add(language)
```

Reply:
Also maybe we lowercase here? e.g. we wouldn't want to warn for JavaScript vs. javascript.

Reply:
Good idea. I will do it.

Reply:
Done @DSchau